R/UploadControls.R

```diff
@@ -605,23 +608,23 @@ join_blood_spot_collections <- function(df.payload, con) {
 #'
 #' @param df.payload A dataframe containing payload data.
 #' @param con A database connection object.
+#' @param count_col Column name in df.payload for count.
+#' @param dbs_control_sheet_id Column name in df.payload for DBS control sheet ID.
 #' @return Result from the dbAppendTable indicating if the collections were added successfully.
-add_new_blood_spot_collections <- function(df.payload, con, count_col) {
-
+add_new_blood_spot_collections <- function(df.payload, con, count_col, dbs_control_sheet_id) {
```

Comment:
I'm a little confused why the count_col and dbs_control_sheet_id are parameterized, do these ever change?